exercises/string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,44 @@
+/* exported capitalizeWords */
+/* input - string
+-output - string
+-create storage space for word
+-create storage space for output
+-look at each character in string
+-capitalize first character
+-lowercase next character
+if character is space :
+-capitalize word with capitalizeWord();
+-add word to output storage space
+-clear word storage space
+-return output storage space */
+function capitalizeWord(word) {
+  var newString = '';
+  newString = newString + word[0].toUpperCase();
+  for (var i = 1; i < word.length; i++) {
+    if (typeof (newString) === 'string') {
+      newString = newString + word[i].toLowerCase();
+    }
+  }
+  if (newString === 'Javascript') {
+    newString = 'JavaScript';
+  }
+  return newString;
+}
```

Comment:
You can remove this, you can still use the function from that other file without having to include it here, since all of your JavaScript is being loaded into the same HTML document